app/Http/Controllers/ApiController.php

```diff
@@ -103,17 +105,30 @@ public function makepayment(Request $request)
                     "name" => $transca->initialize->name,
                     "notify_url" => $transca->initialize->notify_url
                 ]); 
-      
```

Comment:
Looking at your make payment function, it handles only MTN, what of Orange make payment? 

Hint
- Write helper functions for each of the PSPs. Then call the helper function when the PSP is chosen to make the payment.

Reply:
ok noted